frontend/src/components/elements/Audio/Audio.test.tsx

```diff
@@ -65,24 +65,15 @@ describe("Audio Element", () => {
     const props = getProps({
       url: "http://localhost:80/media/sound.wav",
     })
+    const useEffect = jest.spyOn(React, "useEffect")
+    const wrapper = mount(<Audio {...props} />)
 
-    const wrapper = shallow(<Audio {...props} />)
```

Comment:
I'm assuming this was needed cause `useEffect` isn't called in shallow. I want to investigate what we can do to test the `audioRef`

Reply:
@kmcgrady Yes, that's what I am also trying to figure out. If we had rendered text, then it would have been easy to test

Reply:
Thank you for simplifying this 💯